pkg/trace/agent/agent.go

```diff
@@ -296,11 +296,22 @@ func (a *Agent) ProcessStats(in pb.ClientStatsPayload, lang, tracerVersion strin
 	in.TracerVersion = tracerVersion
 	in.Lang = lang
 	for _, group := range in.Stats {
+		var blocked bool
+		i := 0
```

Comment:
```suggestion
		var (
            n       int
            blocked bool
      )
```
This `i` seems more like an `n` 🙂